history-deleter/README.md

```diff
@@ -0,0 +1,15 @@
+# History deleter
+
+## What it does
+
+This extension includes a browser action with a popup specified as "history.html".
+
+The popup shows a list of 5 history entries for the current domain. It provides a clear button to delete all entries for that domain.
+
+## What it shows
+
+How to use the history API.
+
+## Caveats
+
+When using it from about:* pages (for example about:debugging) you have no hostname so that does a history query for everything. 
```

Comment:
It might be worth pointing out here, or in a code comment, that this also means that clicking the clear link will also delete *everything*.